### DISCUSSION TITLE
Add OpenRouter support

### DISCUSSION THREAD

**Discussion by alr86** [Taewan-P/gpt_mobile#131] [Q&A] [answered]
When I use openrouter API as OpenAI, with every message app says:
```
Error: Illegal input: Unexpected JSON token at offset 9: Expected start of the object '{', but had '"' instead at path: $.error
JSON input: {"error":"Not Found"}
```

**Comment by Taewan-P** [comment, marked as answer]
@alr86 Check if your custom API address is correctly set. It should be like this: https://github.com/Taewan-P/gpt_mobile/discussions/62#discussioncomment-10928255

**Reply by alr86** [reply]
Thank you for your response.
I was using ‍`https://openrouter.ai/api/v1/chat/completions`, but the correct one is `https://openrouter.ai/api/v1/`.
